models/stg_sap.yml

```diff
@@ -0,0 +1,2158 @@
+        description: Time/measurement unit
+      - name: ein02
+        description: Time/measurement unit
+      - name: ein03
+        description: Time/measurement unit
+      - name: ein04
+        description: Time/measurement unit
+      - name: ein05
+        description: Time/measurement unit
+      - name: ein06
+        description: Time/measurement unit
+      - name: ein07
+        description: Time/measurement unit
+      - name: ein08
+        description: Time/measurement unit
+      - name: ein09
+        description: Time/measurement unit
+      - name: ein10
+        description: Time/measurement unit
+      - name: ein11
+        description: Time/measurement unit
+      - name: ein12
+        description: Time/measurement unit 
+      - name: ein13
+        description: Time/measurement unit
+      - name: ein14
+        description: Time/measurement unit
+      - name: ein15
+        description: Time/measurement unit
+      - name: ein16
+        description: Time/measurement unit
+      - name: ein17
+        description: Time/measurement unit
+      - name: ein18
+        description: Time/measurement unit
+      - name: ein19
+        description: Time/measurement unit
+      - name: ein20
+        description: Time/measurement unit
+      - name: ein21
+        description: Time/measurement unit
+      - name: ein22
+        description: Time/measurement unit
+      - name: ein23
+        description: Time/measurement unit
+      - name: ein24
+        description: Time/measurement unit
+      - name: ein25
+        description: Time/measurement unit
+      - name: ein26
+        description: Time/measurement unit
+      - name: ein27
+        description: Time/measurement unit
+      - name: ein28
+        description: Time/measurement unit
+      - name: ein29
+        description: Time/measurement unit
+      - name: ein30
+        description: Time/measurement unit
+      - name: ein31
+        description: Time/measurement unit
+      - name: ein32
+        description: Time/measurement unit
+      - name: ein33
+        description: Time/measurement unit
+      - name: ein34
+        description: Time/measurement unit
+      - name: ein35
+        description: Time/measurement unit
+      - name: ein36
+        description: Time/measurement unit 
+      - name: ein37
+        description: Time/measurement unit
+      - name: ein38
+        description: Time/measurement unit
+      - name: ein39
+        description: Time/measurement unit
+      - name: ein40
+        description: Time/measurement unit
+      - name: falgk
+        description: Case group catalog
+      - name: falgr
+        description: Case group
+      - name: flag1
+        description: Reserved field/unused field
+      - name: flag2
+        description: Reserved field/unused field
+      - name: flag3
+        description: Reserved field/unused field
+      - name: flag4
+        description: Reserved field/unused field
```

Comment:
Are these all intended to be the same description?

Reply:
Yes 
<img width="751" alt="Screenshot 2023-07-06 at 4 25 12 AM" src="https://github.com/fivetran/dbt_sap_source/assets/108772760/5f05e614-2a67-4f39-8635-9e1c9fdc295f">